data/strings/strings.txt

```diff
@@ -23342,54 +23248,54 @@
-    el = Να ειδοποιούμαι αν υπάρχει κίνδυνος να υπερβώ το όριο ταχύτητας
-    es = Alertar si existe el riesgo de exceder el límite de velocidad
-    es-MX = Avisar si existe el riesgo de exceder el límite de velocidad
+    comment = "Speed cameras" settings menu option (should be short! no more than 47-50 chars) to warn a driver if there is a risk of exceeding the speed limit
+    tags = android,ios
+    en = Warn if exceeding the speed limit
+    af = Waarsku teen spoed
+    ar = التحذير من السرعة
+    az = Sürətlə bağlı xəbərdarlıq edin
+    be = Папярэджваць калі хуткасць перавышана
+    bg = Да ви предупреди за превишена скорост
+    ca = Avisa’m en excedir velocitat límit
+    cs = Varování před překročením rychlosti
+    da = At advare dig om hastighedsoverskridelser
+    de = Um dich vor Geschwindigkeitsüberschreitungen zu warnen
+    el = Για να σας προειδοποιήσει για υπερβολική ταχύτητα
+    es = Alertar al exceder velocidad límite
+    es-MX = Avisar al exceder velocidad límite
     et = Hoiatused kui on oht ületada piirkiirust
-    eu = Ohartarazten du abiadura muga gainditzeko arriskua badago
-    fa = در صورتی که خطر تجاوز از سرعت مجاز وجود داشت، نسبت هشدار دهد
-    fi = Varoita jos tuntuu siltä, että nopeusrajan ylitys on mahdollista
-    fr = Alerte sonore en cas de risque de dépassement de la limite de vitesse
-    he = הזהר אם יש חשש להפרזה במהירות
-    hu = Figyelmeztetés ha van kockázat a sebesség korlátozás megsértéséről
-    id = Peringatkan saat terdapat risiko melebihi batas kecepatan
-    it = Avvisare in caso di rischio eccesso velocità
-    ja = 速度制限超過の危険性がある場合には、自動速度違反取締装置について警告します
-    ko = 속도 제한 초과 위험이 있을때 경고하기
-    lt = įspėti jei yra greičio viršijimo rizika
-    mr = वेगमर्यादा ओलांडण्याचा धोका असल्यास चेतावणी द्या
-    nb = Advarsel hvis det er fare for å overskride fartsgrensen
-    nl = Waarschuw als er een risico bestaat dat de snelheidslimiet wordt overschreden
-    pl = Ostrzegaj jeśli istnieje ryzyko przekroczenia ograniczenia prędkości
-    pt = Avisar se houver risco de exceder o limite de velocidade
-    pt-BR = Avisar se houver risco de ultrapassar o limite de velocidade
-    ro = Avertizează dacă există riscul depășirii limitei de viteză
-    ru = Предупреждать если есть риск превышения скоростного лимита
-    sk = Upozornenie ak existuje riziko prekročenia rýchlostného limitu
-    sv = Varna om det finns risk att överskrida hastighetsgränsen
-    sw = Onyo iwapo kuna hatari yoyote ya kuzidi ukomo wa mwendo-kasi
-    th = เตือนเกี่ยวกับกล้องตรวจจับความเร็วหากมีความเป็นได้ว่าจะขับเร็วเกิดกำหนด
+    eu = Abiaduraz ohartarazi
+    fa = هشدار در مورد سرعت غیر مجاز
+    fi = Varoittaa sinua ylinopeudesta
+    fr = Avertir d'un excès de vitesse
+    he = התריע על מהירות מופרזת
+    hi = तेज़ गति से गाड़ी चलाने के बारे में चेतावनी दें
+    hu = Gyorshajtási figyelmeztetés
+    id = Untuk memperingatkan Anda agar tidak ngebut
+    it = Per avvisarti di un eccesso di velocità
```

Comment:
https://github.com/organicmaps/organicmaps/pull/7585#issuecomment-1996602756 by @lucaceccarini:

```suggestion
    it = Avvisa al superamento del limite di velocità
```

@lucaceccarini can you choose between the two?

Reply:
Since the proposed string is now longer than the original, I do prefer the more succinct variation

`it = Avvisa se superi il limite di velocità`

Reply:
thanks, updated the comment

Reply:
@lucaceccarini: Please review:

![ios_speed_cam_prefs_it1](https://github.com/organicmaps/organicmaps/assets/25888296/0d456481-a115-48cd-bbea-6304807660ae)

Reply:
OK, for me @gpesquero.